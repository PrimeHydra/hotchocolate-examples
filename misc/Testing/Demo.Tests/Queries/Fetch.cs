using System.Text;
using Microsoft.Extensions.DiagnosticAdapter.Internal;

namespace Demo.Tests.Queries;

public class Fetch : QueryTestBase
{
    [Fact]
    public async Task FetchSophie()
    {
        var result = await ExecuteQueryAsync(
            @"
                {
                    puppy(id: ""Id_Sophie"") { 
                        name 
                        breed 
                    } 
                }
                ");

        result.MatchSnapshot();
    }
}