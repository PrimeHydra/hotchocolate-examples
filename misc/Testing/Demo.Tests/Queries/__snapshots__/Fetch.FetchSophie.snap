{
  "data": {
    "puppy": {
      "name": "Sophie D Chillingshead",
      "breed": "Chihuahua"
    }
  }
}
